include/sound/sof/dai-intel.h

```diff
@@ -83,7 +83,7 @@ struct sof_ipc_dai_ssp_params {
 /* HDA Configuration Request - SOF_IPC_DAI_HDA_CONFIG */
 struct sof_ipc_dai_hda_params {
 	struct sof_ipc_hdr hdr;
-	/* TODO */
```

Comment:
we need to remove the hdr, it's not needed. Maybe do this in a separate patch for API change on firmware and driver at the same time.